src/main/java/org/team1540/robot2019/OI.java

```diff
@@ -125,7 +125,7 @@ public static void initButtons() {
         prepareToClimbButton.whenPressed(new PrepareForClimb());
         startClimbingButton.whenPressed(new SimpleCommand("Climb Safe", () -> {
             if (startClimbingSafety.get()) {
-                new RaiseUpGyroAssist().start();
+                new ClimbLevelThreeGyro().start();
```

Comment:
tbh I think we should just rename to `Level3Climb` since we don't have the percent-based climb anymore

Reply:
Sure, no numbers in classes though